html/base/link.stories.js

```diff
@@ -47,35 +44,161 @@ export const simple = () => (
     >
       Simple Link
     </a>
+  `;
+
+export const light = () =>
+  `
+    <div class="sprk-o-Box" style="background-color: #1C1B1A"> 
```

Comment:
See if we can remove the sprk box here and add to decorator for this individual story.